media-sound/jack/jack-4.0.0_pre20200706.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_7 )
```

Comment:
Our policy right now is "as many Pythons (3.x) as possible", so 3.6, 3.7, 3.8, 3.9 if it works/tests pass with all, please!

Reply:
Hello @thesamesam , thanks for your comments, you're the fastest reviewer I have seen as proxy-maintainer.

Let me say that I know some of your comments, I want mainly indicate that I am working on this ebuild.
I will test as many python versions as I can, and update the ebuild accordingly.

Reply:
I tested python 3.6 3.7 3.8 and found them working.

Reply:
Fab.